pytab/hwi.py

```diff
@@ -19,14 +19,13 @@
 ##########################################################################################
 import platform
 import subprocess
-from json import dumps, loads
+import json
```

Comment:
Because seeing json.loads(something) and json.dumps(something) in the code is much easier to see what it is doing than simply loads(something) and dumps(something)

Reply:
This code doesnt do the same though.
We are now importing all features of `json`, even though we dont use them.

This doesnt REALY matter, but it was done like this by choice,